rust/src/sdk/api.rs

```diff
@@ -242,33 +334,41 @@ macro_rules! get_model {
             #[doc = "to the latest global model.\n"]
```

Comment:
nit: I think we could make this a bit less noisy with a raw string literal in a single `#[doc]` attribute. Like:

```rust
#[doc = r#"some lines
another line
etc.
"#]
```

Reply:
good point, i didn't know that this is possible. it is indeed quite ugly to read due to the mixture of macros and docs.

Reply:
unfortunately, i didn't find a way to make it work. raw string literals don't respect any of the markdown and linebreaks (this is of course intended, it is a raw string literal after all) and using just common string literals triggers some weird markdown formatting (code blocks etc). i guess we have to stick with the ugly one doc attribute per line.